src/main/java/uk/ac/ebi/atlas/search/GeneSearchDao.java

```diff
@@ -138,58 +143,53 @@ public GeneSearchDao(NamedParameterJdbcTemplate namedParameterJdbcTemplate,
         );
     }
 
-    // Returns inferred cell types and organism parts for each experiment accession
-    public Map<String, Map<String, List<String>>>
-    getFacets(List<String> cellIds, SingleCellAnalyticsSchemaField... singleCellAnalyticsSchemaFields) {
-        List<SingleCellAnalyticsSchemaField> subFacetFields = Arrays.asList(singleCellAnalyticsSchemaFields);
+    // Returns all the metadata values for each experiment accession, given a subset of metadata types
+    public Map<String, Map<String, List<String>>> getFacets(List<String> cellIds, String... metadataTypes) {
+        List<String> facetFieldValues = Arrays.asList(metadataTypes);
```

Comment:
Instead of this, I think inlining `ImmutableList.of(metadataTypes)` below in the `queryBuilder` is less distracting.